vanilla.transform.cjs

```diff
@@ -0,0 +1,12 @@
+/* eslint-env node */
+/**
+ * @file Re-exports the vanilla-extract jest transform, so that jest can properly transform .css.ts files.
+ * @vanilla-extract/jest-transform incorrectly maps its default export, so that `import *` does not work; and expects
```

Comment:
Would annotate this with ` so that `@vanilla` is not highlighted as JSDoc component.